geowebcache/s3storage/src/main/java/org/geowebcache/s3/S3BlobStoreConfig.java

```diff
@@ -301,4 +301,16 @@ public BlobStore createInstance(TileLayerDispatcher layers, LockProvider lockPro
         return new S3BlobStore(this, layers, lockProvider);
     }
 
+    @Override
+    public String getLocation() {
+        String bucket = this.getBucket();
+        String prefix = this.getPrefix();
+        if(prefix==null){
+            return String.format("bucket: %s", bucket);
+        } else {
+            return String.format("bucket: %s prefix: %s", bucket, prefix);
+        }
+        
```

Comment:
@groldan Does this seem like a reasonable way to indicate the location of the store in a human readable form?